app/src/main/java/com/hedvig/app/feature/home/ui/HomeFragment.kt

```diff
@@ -39,6 +39,7 @@ class HomeFragment : Fragment(R.layout.home_fragment) {
     override fun onResume() {
         super.onResume()
         loggedInViewModel.onScroll(scroll)
+        model.load()
```

Comment:
Will this trigger a reload everytime we switch to this tab?

Reply:
Fixed to only trigger on VM instantiation and when getting back the result of the embark activity